IBPSA/Fluid/HeatPumps/ModularReversible/Modular.mo

```diff
@@ -0,0 +1,136 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible;
+model Modular
+  "Grey-box model for reversible heat pumps"
+  extends
+    IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.PartialReversibleRefrigerantMachine(
+    final use_COP=true,
+    final use_EER=use_rev,
+    con(preDro(m_flow(nominal=QHea_flow_nominal/1000/10))),
+    eva(preDro(m_flow(nominal=QHea_flow_nominal/1000/10))),
+    final PEle_nominal=refCyc.refCycHeaPumHea.PEle_nominal,
+    mCon_flow_nominal=QHea_flow_nominal/(dTCon_nominal*cpCon),
+    mEva_flow_nominal=(QHea_flow_nominal - PEle_nominal)/(dTEva_nominal*cpEva),
+    use_rev=true,
+    redeclare IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.RefrigerantCycle refCyc(
+      redeclare model RefrigerantCycleHeatPumpHeating =
+          RefrigerantCycleHeatPumpHeating,
+      redeclare model RefrigerantCycleHeatPumpCooling =
+          RefrigerantCycleHeatPumpCooling));
+  parameter Modelica.Units.SI.HeatFlowRate QHea_flow_nominal(min=Modelica.Constants.eps)
+    "Nominal heating capacity"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.HeatFlowRate QCoo_flow_nominal(max=0)=0
+    "Nominal cooling capacity"
+      annotation(Dialog(group="Nominal condition - Cooling", enable=use_rev));
+
+  replaceable model RefrigerantCycleHeatPumpHeating =
+    IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.BaseClasses.PartialHeatPumpCycle
+      (PEle_nominal=0)
+       constrainedby
+    IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.BaseClasses.PartialHeatPumpCycle(
+       final useInHeaPum=true,
+       final QHea_flow_nominal=QHea_flow_nominal,
+       final TCon_nominal=TConHea_nominal,
+       final TEva_nominal=TEvaHea_nominal,
+       final cpCon=cpCon,
+       final cpEva=cpEva)
+  "Refrigerant cycle module for the heating mode"
+    annotation (choicesAllMatching=true);
+
+  replaceable model RefrigerantCycleHeatPumpCooling =
+      IBPSA.Fluid.Chillers.ModularReversible.RefrigerantCycle.BaseClasses.NoCooling
```

Comment:
This is the replaceable model. The default is NoCooling, so the user can just set use_rev=false and simulate the model. If we use the partial one, the user would have to change the model for reversible operation and the boolean. Due to the enable-annotation, that can be frustrating.
This is also linked to the comment about the partial modular model.

Reply:
So, the model is suited for non-reversible HP by setting `use_rev=false`.
Since the default is `NoCooling ` then `use_rev` should be set to `false` by default so we have a consistent set of default settings, no?

Furthermore, the model documentation includes:
> Grey-box model for reversible heat pumps
Model of a reversible, modular heat pump

and the package is `ModularReversible` with:

> Package for the modular reversible heat pumps
Package with models and packages for the modular reversible heat pump approach

If non-reversible HP can be modeled as well, than this needs to be updated.

Reply:
Agree, I update the doc and changed the default.